packages/web-components/fast-element/src/context.spec.ts

```diff
@@ -280,7 +280,7 @@ describe("Context", () => {
             const elementName = uniqueElementName();
 
             class TestElement extends HTMLElement {
-                @TestContext test: string;
+                @TestContext test!: string;
```

Comment:
Okay so 
```suggestion
                @TestContext test = '';
```
wasn't working in some cases. Sounds like it changes the behaviour of the property and make it impossible to set a value to it after that. There might be a better way, if so feel free to point me to the right direction and I'll make that change 👌